public/index.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html lang="en-us">
+<head>
+  <title>avro-schema-registry</title>
+  <style type="text/css">
+    body {
+      background-color: #444;
+      color: #FFF;
+      font-family: "proxima-nova", helvetica, verdana, arial, sans-serif;
+      margin: 4%;
+      padding: 16px;
+      font-size: 1.75em;
+      text-rendering: optimizeLegibility;
+      font-feature-settings: "kern" 1;
+      font-kerning: normal;
+    }
+    h1 {
+      margin: 1.5em 0 0;
+    }
+    p {
+      font-weight: lighter;
+      line-height: 120%;
+    }
+    a {
+      color: inherit;
+    }
+    #container {
+      padding:10px;
+    }
+    .content {
+      padding-top:0em;
+    }
+  </style>
+</head>
+<body>
```

Comment:
Can you catch up with @jredburn on any additional branding we want to provide here (e.g. a Salsify logo)?